src/aws/terraform_helpers.go

```diff
@@ -43,7 +44,7 @@ func createRemoteState(currSession *session.Session, config *aws.Config, bucketN
 }
 
 // creates dynamodb table to store terraform lock file
-func createLockTable(currSession *session.Session, config *aws.Config, tableName string) error {
+func createLockTable(currSession client.ConfigProvider, config *aws.Config, tableName string) error {
```

Comment:
What was the warning for this just out of curiosity?

Reply:
That `*session.Session` is more specific than the requirements of its use in this function.